torchvision/datapoints/_dataset_wrapper.py

```diff
@@ -78,8 +82,21 @@ def wrap_dataset_for_transforms_v2(dataset):
 
     Args:
         dataset: the dataset instance to wrap for compatibility with transforms v2.
+        target_keys: Target keys to return in case the target is a dictionary. If ``None`` (default), selected keys are
+            specific to the dataset. See above for details. If ``"all"``, returns the full target. Can also be a
+            collection of strings for fine grained access.
     """
-    return VisionDatasetDatapointWrapper(dataset)
+    if not (
+        target_keys is None
+        or target_keys == "all"
+        or (isinstance(target_keys, collections.abc.Collection) and all(isinstance(key, str) for key in target_keys))
+    ):
+        raise ValueError(
+            f"`target_keys` can be None, 'all', or a collection of strings denoting the keys to be returned, "
+            f"but got {target_keys}"
+        )
```

Comment:
Do we also want to raise an error if `target_keys` is passed for any dataset that isn't `Coco`, `WIDERFace`, etc. ?

Reply:
Yeah, let's play safe here.